Cartfile.resolved

```diff
@@ -1,4 +1,4 @@
-binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" "5.2.0"
+binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" "5.5.0"
```

Comment:
I think I must’ve either hand-edited this file or run it while experimenting with an `==` operator in the Cartfile, but the tadpole operator would actually resolved to v5.6.1. Fixed in #2317.